#
# On Mac devices, noVNC remaps the Command keys to Alt keys. It also remaps Alt
# keys to AltGraph keys. Disable these remappings since they don't make sense
# when controlling Linux applications. Instead, let the X/VNC server handle
# needed remappings. For example, it can remap Command keys to Control keys (for
# copy/paste shortcuts).
#
--- a/core/input/keyboard.js
+++ b/core/input/keyboard.js
@@ -142,7 +142,7 @@
         // keys around a bit to make things more sane for the remote
         // server. This method is used by RealVNC and TigerVNC (and
         // possibly others).
-        if (browser.isMac() || browser.isIOS()) {
+        if (false) {
             switch (keysym) {
                 case KeyTable.XK_Super_L:
                    keysym = KeyTable.XK_Alt_L;
